### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Elijahdre**
- Add Media Query to the styling.
- Add desktop background image
- Add Linters
- Add Readme file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -549,6 +549,25 @@ textarea {
...
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  /* header section */
   .hero-section {
     background-image: url(assets/images/desktop-bg-img.png);
     background-repeat: no-repeat;
```
Comment (mirouhml) [inline style.css:573]: - [ ] Please add more padding to the top of your `hero-section` it's too compact, kindly use the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) to decide how much to add. 😁

File: style.css [hunk 2]
```diff
@@ -561,6 +580,26 @@ textarea {
     width: 70%;
   }
 
+  /* work section */
+  .container {
+    display: flex;
+    flex-direction: row;
+    margin: 30px 142px;
+  }
```
Comment (mirouhml) [inline style.css:588]: - [ ] The images in the project cards should be alternated like the first project should have the project picture on the **left**, the one under it on the **right**, and then the one under it on the **left**, and so on, like the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848), kindly change it to match that. 😁

image

File: style.css [hunk 3]
```diff
@@ -561,6 +580,26 @@ textarea {
...
+
+  .Project-one-header {
+    width: 100%;
+  }
+
+  /* About section  */
   .about-content {
     display: flex;
     flex-direction: row;
```
Comment (mirouhml) [inline style.css:605]: - [ ] The content in the about section, should match the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848):

Right now your design looks like this:
image

And the one on Figma is like this:
image

Kindly fix it so they match 😁

File: style.css [hunk 4]
```diff
@@ -583,10 +622,11 @@ textarea {
...
- .about-description {
+  .about-description {
     font-size: 15px;
-  } 
+  }
 
+  /* Contact section */
   .contact-me {
     background-image: url(assets/images/desktop-bg-img.png);
   }
```
Comment (mirouhml) [inline style.css:632]: - [ ] Contact section has a lot of padding on top but almost none at the bottom, they should be evened out, kindly do that, please use the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) to design how much to add for the top and for the bottom. 😁

This is your design right now:

image

And this is the Figma design:

image

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Elijahdre, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -549,18 +549,62 @@ textarea {
...
+  }
+
+  .Project-one-header {
+    width: 100%;
+  }
+
+  /* About section  */
   .about-content {
     display: flex;
     flex-direction: row;
```
Comment (mirouhml) [inline style.css:610]: - [ ] As I mentioned in my previous review, kindly fix the design of the about myself section to match the design of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848). 😁

Right now the whole section is only using half of the area, it should use the whole width, this is how it looks for me:
image

And this is how the Figma design looks:
image

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @Elijahdre, 

Great work on making the changes I requested earlier 👏 

You've done well implementing some of them, but there is still one small issue left, but don't worry you're almost there!! 😉

image

 ## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Elijahdre , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Elijahdre) [reply style.css:573]: @mirouhml Thank you for this review. I will do that now.

File: style.css (hunk 3 above)
Comment (Elijahdre) [reply style.css:610]: @mirouhml I will make it match now. Thank you🙏

File: style.css (hunk 4 above)
Comment (Elijahdre) [reply style.css:651]: @mirouhml Thank you for this review.

File: style.css (hunk 5 above)
Comment (Elijahdre) [reply style.css:610]: @mirouhml Thank you very much. It just skipped my remembrance.

**Comment by Elijahdre** [comment]
@MeqdamAlqudah Thank you for the approval.

**Pull request merged**